fabric-api-base/src/main/resources/fabric.mod.json

```diff
@@ -5,5 +5,8 @@
   "license": "Apache-2.0",
   "depends": {
     "fabricloader": ">=0.4.0"
-  }
+  },
+  "mixins": [
```

Comment:
Why change the base fabric.mod.json ? This seems like a mistake.

Reply:
This is marked as resolved but it doesn't seem to be? Keep the description